### ISSUE TITLE
Interesting

### ISSUE THREAD

**Issue by mattross1** [CIST2100Project/BusinessApplication#14]
While I don't have much experience with RPG related games, I'm curious to what your financial strategy would be. Once again with little experience in this area, I personally don't see the appeal of something like this, especially indie developed. There seem to be many more exciting RPG games to play. While there are undoubtedly people who may enjoy such a game, the premise of the game seems extremely extremely niche making it hard to see a return on investment. Indie game development is not an easy place to be and can easily tank your money and time, without a concept or premise 100% proven to have a large audience, debt can easily pile up.

**Comment by Kpalmquist** [comment]
Yes, what can you use to attract your consumers? The description of the game is interesting, but after the completion of the game what will be the most known reason to acquire your game?
